rosdep/python.yaml

```diff
@@ -3018,6 +3018,11 @@ python-pathlib:
   debian: [python-pathlib]
   gentoo: [dev-python/pathlib]
   ubuntu: [python-pathlib]
+python-pathlib2:
+  arch: [python2-pathlib2]
+  debian: [python-pathlib2]
+  gentoo: [dev-python/pathlib2]
+  ubuntu: [python-pathlib2]
```

Comment:
I think xenial needs to be marked out with `xenial: null` since that package doesn't exist for xenial

Reply:
Yes, `python-pathlib2` wasn't released on xenial. Would it better to install from `pip` rather than the package manager? e.g.
```yaml
ubuntu:
      pip:
           pathlib2
```

Reply:
No, we prefer to use the OS package manager. We should either mark Xenial with `null` or add a separate top-level key for installing from pip (e.g. `python-pathlib2-pip`). I think marking xenial is enough, unless you really want an option for Xenial.